b926173f broke compilation on macos

The change in b926173f breaks compilation on macos

```
config.status: executing depfiles commands
  CC       inadyn-main.o
main.c:161:11: error: use of undeclared identifier 'L_cuserid'
        char buf[L_cuserid];
                 ^
main.c:164:8: warning: implicit declaration of function 'cuserid' is invalid in C99 [-Wimplicit-function-declaration]
                if (!cuserid(buf))
                     ^
main.c:452:7: warning: 'daemon' is deprecated: first deprecated in macOS 10.5 - Use posix_spawn APIs instead. [-Wdeprecated-declarations]
                if (daemon(0, 0) < 0) {
                    ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/stdlib.h:292:6: note: 'daemon' has been explicitly marked deprecated here
int      daemon(int, int) __DARWIN_1050(daemon) __OSX_AVAILABLE_BUT_DEPRECATED_MSG(__MAC_10_0, __MAC_10_5, __IPHONE_2_0, __IPHONE_2_0, "Use posix_spawn APIs instead.") __WATCHOS_PROHIBITED __...
         ^
2 warnings and 1 error generated.
make[1]: *** [inadyn-main.o] Error 1
make: *** [all-recursive] Error 1
```

--- a/src/main.c
+++ b/src/main.c
@@ -21,6 +21,7 @@
  */
 
 #include <getopt.h>
+#include <stdio.h>
 #include <stdlib.h>
 #include <pwd.h>		/* getpwnam() */
 #include <grp.h>		/* getgrnam() */

